.github/workflows/release.yml

```diff
@@ -5,9 +5,25 @@ on:
     tags: ["v*"]
 
 jobs:
+  npm_publish:
+    name: Publish to npm
+    runs-on: ubuntu-latest
+
+    if: github.actor == "nicolo-ribaudo"
+
+    steps:
+      - name: Checkout the new tag
+        uses: actions/checkout@v1.0.0
+      
```

Comment:
Consider move `Check if releasing from master` step to here now that `npm_publish` will run _before_ `github_release`.

Reply:
I think that if we publish a v* tag we would always want to publish it to npm (for example, for a special version), but we might not want to add it to the changelog.

Reply:
I wound rather be conservative here, we could add branch to whitelist if publishing special version. But we should prevent if people push their local tags in their own branch and it ends up with a public npm release.